monitoring/storage_linux.go

```diff
@@ -143,8 +147,13 @@ func (c *storageChecker) checkFsType(ctx context.Context, reporter health.Report
 	return nil
 }
 
+// checkHighWatermark checks the disk usage. A failed warning or critical probe
+// will be reported if the usage percentage is above the set thresholds.
+// If the WatermarkWarning percentage is higher than or equal to the
```

Comment:
Does it make sense to be able to set warning values exceeding the critical value?